examples/text-generation/run_generation.py

```diff
@@ -91,6 +91,12 @@ def setup_parser(parser):
         type=str,
         help="If `--dataset_name` was given, this will be the name of the column to use as prompts for generation.",
     )
+    parser.add_argument(
+        "--use_cache",
```

Comment:
what is this argument for?

Reply:
Removed parameter since it is being covered under use_kv_cache